zebedee-cms/src/main/java/com/github/onsdigital/zebedee/api/Root.java

```diff
@@ -126,15 +85,61 @@ public static void init() {
             Collections.CollectionList collections = zebedee.getCollections().list();
             alertOnInProgressCollections(collections, notifier);
             loadExistingCollectionsIntoScheduler(collections);
-        } catch (IOException e) {
-            final String message = "failed to load collections list on startup";
-            error().logException(e, message);
-            throw new RuntimeException(message, e);
+        } catch (IOException ex) {
+            throw new RuntimeException("failed to load collections list on startup", ex);
         }
 
         info().data(ZEBEDEE_ROOT, rootDir).log("zebedee cmd initialization completed successfully");
     }
 
+    /**
+     * Validate the zebedee root path meets the requirements - not null/empty, the file exists and is a directory.
+     * Throws {@link RuntimeException} if any of the checks fail.
+     *
+     * @param rootDir the path to validate.
+     * @return the root dir value as a {@link Path} object.
+     */
+    private static Path validateZebedeeRootConfig(String rootDir) {
+        if (StringUtils.isEmpty(rootDir)) {
```

Comment:
This check does not seem to get called, see 

ReaderConfiguration line 105

Reply:
ReaderConfiguration line 153 assigns it to the class field which is then used in the subsequent constructor logic.